src/boxed.tsx

```diff
@@ -13,16 +13,17 @@ type StylesProps = {
 
 const useStyles = createUseStyles(({colors, isDarkMode}) => ({
     boxed: {
-        background: ({isInverseInside}: StylesProps) =>
+        backgroundColor: ({isInverseInside}: StylesProps) =>
             isInverseInside && !isDarkMode ? colors.backgroundBrand : colors.backgroundContainer,
+        backgroundClip: ({isInverseInside}: StylesProps) => (isInverseInside ? 'padding-box' : 'border-box'), // for transparent borders, no background is drawn beneath the border.
 
         border: ({isInverseOutside, isInverseInside}: StylesProps) => {
             if (isInverseOutside && !isInverseInside) {
                 return `1px solid ${colors.backgroundContainer}`;
             }
 
-            if (isInverseInside && !isDarkMode) {
```

Comment:
I think the border must be transparent when isInverseInside in both light and dark mode, but I'm not sure